### DISCUSSION TITLE
Asset Balance and Asset Transactions

### DISCUSSION THREAD

**Discussion by ghost** [CareBoo/unity-algorand-sdk#136] [Q&A] [answered]
Would appreciate some details on how to show Asset Balance and Asset Transactions using the AccountInfo.

`[AlgoApiField("assets", null, false)]
public AssetHolding[] Assets`

Thanks!

**Comment by jasonboukheir** [comment, marked as answer]
Hi,

The following methods from the indexer should give you a good starting point:
- [LookupAssetBalances](https://careboo.github.io/unity-algorand-sdk/4.0/api/Algorand.Unity.IndexerClient.html#Algorand_Unity_IndexerClient_LookupAssetBalances_System_UInt64_Algorand_Unity_Optional_System_Boolean__Algorand_Unity_Optional_System_UInt64__System_String_Algorand_Unity_Optional_System_UInt64__Algorand_Unity_Optional_System_UInt64__)
- [LookupAssetByID](https://careboo.github.io/unity-algorand-sdk/4.0/api/Algorand.Unity.IndexerClient.html#Algorand_Unity_IndexerClient_LookupAssetByID_System_UInt64_Algorand_Unity_Optional_System_Boolean__)
- [LookupAssetTransactions](https://careboo.github.io/unity-algorand-sdk/4.0/api/Algorand.Unity.IndexerClient.html#Algorand_Unity_IndexerClient_LookupAssetTransactions_System_UInt64_System_Byte___Algorand_Unity_Optional_System_Boolean__System_String_Algorand_Unity_Optional_System_UInt64__DateTime_System_String_Algorand_Unity_Optional_System_Boolean__Algorand_Unity_Optional_System_UInt64__DateTime_Algorand_Unity_Optional_System_UInt64__Algorand_Unity_Optional_System_UInt64__Algorand_Unity_Address_Algorand_Unity_Optional_System_UInt64__Algorand_Unity_SignatureType_System_String_Algorand_Unity_AddressRole_Algorand_Unity_Optional_System_UInt64__)
